torch/onnx/symbolic_opset9.py

```diff
@@ -717,11 +717,19 @@ def squeeze(g, self, dim=None):
 
 def prelu(g, self, weight):
     self_rank = sym_help._get_tensor_rank(self)
-    if self_rank is not None and self_rank > 2:
-        weight = sym_help._unsqueeze_helper(g, weight, list(range(1, self_rank - 1)))
+    if self_rank is not None:
+        if self_rank > 2:
+            weight = sym_help._unsqueeze_helper(g, weight, list(range(1, self_rank - 1)))
+        elif self_rank == 0:
+            self = sym_help._unsqueeze_helper(g, self, [0])
+            self_rank = sym_help._get_tensor_rank(self)
+
+    slope_rank = sym_help._get_tensor_rank(weight)
```

Comment:
Don't mix torch names and ONNX names, it's confusing.

```suggestion
    weight_rank = sym_help._get_tensor_rank(weight)
```